apps/src/templates/progress/ProgressLessonTeacherInfo.jsx

```diff
@@ -84,8 +84,10 @@ class ProgressLessonTeacherInfo extends React.Component {
       isLessonHiddenForSection(hiddenLessonState, sectionId, lesson.id);
     const courseId =
       (section && section.code && parseInt(section.code.substring(2))) || null;
-    const loginRequiredLessonStartUrl =
-      lesson.lessonStartUrl + '?login_required=true';
+    const loginRequiredLessonStartUrl = new URL(
```

Comment:
Oh, nice!  Maybe put a comment here indicating that we need to do this because lesson.lessonStartUrl is protocol-relative?